bittensor/_dendrite/dendrite_impl.py

```diff
@@ -452,7 +452,7 @@ def forward_text(
             raise ValueError(error_msg)
             
         # Check list types.
-        if not isinstance(inputs[0], torch.LongTensor):
+        if not ( isinstance(inputs[0], torch.cuda.LongTensor) or isinstance(inputs[0], torch.LongTensor)) :
```

Comment:
Why the cuda catch here?

Reply:
Does this not catch cuda tensors?

Reply:
Yea, so this was one of the main issues, I ran into. In torch, cuda tensors and cpu have different instances and need to be separated.